crit/explore.go

```diff
@@ -145,6 +145,11 @@ func (c *crit) ExploreFds() ([]*Fd, error) {
 			Path: filePath,
 		})
 
+		// Omit if the process has no file descriptors
+		if len(fdEntry.Files) == 0 {
+			continue
+		}
```

Comment:
This change is not directly related to the renaming of `fds|mems` to `fd|mem`. It should be in a separate commit.